modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/gmos/GmosControllerEpics.scala

```diff
@@ -282,4 +291,8 @@ object GmosControllerEpics {
     case InBeam    => "IN-BEAM"
   }
 
+  val defaultTimeout: Time = Seconds(60)
```

Comment:
Usually constants have capitalized casing, like `DefaultTimeout`

Reply:
OK, I'll change it.